lib/snapshot.js

```diff
@@ -52,7 +52,9 @@ var default_UA         = '';
 
 async function init_browser() {
 	const args = process.env.MSHOTS_CONTAINERIZED ? [ '--no-sandbox' ] : [];
-	browser = await _puppeteer.launch( { headless: true, timeout: 30000, ignoreHTTPSErrors: true, args } ).
+  const executablePath = process.env.ARCH === "arm64" ? "chromium" : "chromium-browser";
```

Comment:
Is Puppeteer not able to detect the executable depending on which browser was installed? (chromium vs. chromium-browser)? Or does it need to be explicitly stated because of the filename differences?

Reply:
Yup it's not able to detect this manually, we'll need to specify the executable. Haven't found a way around this!